settings.py

```diff
@@ -897,6 +898,8 @@
 SENTRY_QUERY_URL = 'https://sentry.io/{org}/{project}/?query='
 SENTRY_API_KEY = None
 
+OBFUSCATE_PASSWORD_FOR_NIC_COMPLIANCE = False
+RESTRICT_USED_PASSWORDS_FOR_NIC_COMPLIANCE = False
```

Comment:
Just that both of these are for different functionalities. Also helped in testing one thing at a time.